03week/ticTacToe.js

```diff
@@ -2,17 +2,25 @@
 
 const assert = require('assert');
 const readline = require('readline');
+
 const rl = readline.createInterface({
   input: process.stdin,
   output: process.stdout
 });
+
 let board = [
   [' ', ' ', ' '],
   [' ', ' ', ' '],
   [' ', ' ', ' ']
 ];
 
 let playerTurn = 'X';
```

Comment:
instead of using global variables, your win methods should return true or false.